src/DynamicData/List/ISourceList.cs

```diff
@@ -12,12 +13,36 @@ namespace DynamicData
     /// as well as data access methods
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    public interface ISourceList<T> : IObservableList<T>
+    public interface ISourceList<T> : IObservableList<T>, IList<T>
     {
         /// <summary>
         /// Edit the inner list within the list's internal locking mechanism
         /// </summary>
         /// <param name="updateAction">The update action.</param>
         void Edit(Action<IExtendedList<T>> updateAction);
+
+        /// <summary>
+        /// Gets the count.
+        /// </summary>
+        new int Count { get; }
+
+        //
+        // Summary:
+        //     Gets or sets the element at the specified index.
+        //
+        // Parameters:
+        //   index:
+        //     The zero-based index of the element to get or set.
+        //
+        // Returns:
+        //     The element at the specified index.
+        //
+        // Exceptions:
+        //   T:System.ArgumentOutOfRangeException:
+        //     index is not a valid index in the ISourceList`1.
+        //
+        //   T:System.NotSupportedException:
+        //     The property is set and the ISourceList`1 is read-only.
```

Comment:
Looks like something went wrong with the XML documentation, probably worth adjusting it a bit.

Reply:
I copied the comment from IList's comment for its indexer property directly, and just swapped `IList` for `ISourceList`.  The odd looking `1 is in the original comments, too, as it's a result of how generics are toString()'ed by default.   I do agree it's an odd way to display generics, but it is standard, I'd say.  I don't mind swapping it out for something we all agree we like better, though.

Reply:
I think @worldbeater is referring it's not in xmldoc format with the `///` and XML tags.

Reply:
In the current form it won't be picked up by intellisense.

Reply:
Ah, my bad.  I'm following now.
Swapping it to the xml format.  Thanks!